Products/Zuul/facades/devicefacade.py

```diff
@@ -201,7 +201,8 @@ def _typecatComponentPostProcess(self, brains, total):
             severities = zep.getWorstSeverity(uuids)
             for r in comps:
                 r.setWorstEventSeverity(severities[r.uuid])
-        return SearchResults(iter(comps), total, hash_, False)
+        sortedComps = list(sorted(comps, key=lambda x: getattr(x, sort), reverse=reverse))
```

Comment:
Is creating a new list object necessary here?  The `sorted` function is already returning a list object.